download_bce_data.py

```diff
@@ -1,41 +1,41 @@
 import csv
-import tempfile
+import os
+import openpyxl
 
-import urllib.request, urllib.parse, urllib.error
-from xlrd import open_workbook
+bank_es = 'REGBANESP_CONESTAB_A.xlsx'
 
-urlfile = urllib.request.URLopener()
-with tempfile.NamedTemporaryFile() as banks:
-    urlfile.retrieve('http://www.bde.es/f/webbde/SGE/regis/ficheros/es/'
-        'REGBANESP_CONESTAB_A.XLS', banks.name)
-    book = open_workbook(banks.name)
-    # We filter by 'bancos, cajas de ahorro, cooperativas de credito, entidades
-    # de credito comunitarias y extracomunitarias'
-    bank_types = ['BP', 'CA', 'CC', 'CO', 'EFC', 'OR', 'SECC', 'SECE']
-    bic_codes = {}
+if not os.path.isfile(bank_es):
+    print('Not found REGBANESP_CONESTAB_A file. Download and save to XLSx (openpyxl)')
+    print('wget http://www.bde.es/f/webbde/SGE/regis/ficheros/es/REGBANESP_CONESTAB_A.XLS')
```

Comment:
Why not create an automatic download?

Reply:
XLS format is not supported by openpyxl. Require to convert XLS to XLSx:

```
Download and save to XLSx (openpyxl)
```

See changeset 953b29882d65faf5a9e6a512e892a72d782969ce that replace message:

```
Download and convert to XLSx file (openpyxl support XLSx)
```